common/test/acceptance/pages/lms/teams.py

```diff
@@ -18,4 +19,9 @@ def is_browser_on_page(self):
 
     def get_body_text(self):
         """ Returns the current dummy text. This will be changed once there is more content on the page. """
-        return self.q(css='.page-content-main').text[0]
+        main_page_content_css = '.page-content-main'
+        EmptyPromise(
+            lambda: len(self.q(css=main_page_content_css).text) == 1,
```

Comment:
Ok. If the test ends up being flaky later on, perhaps a deeper dive into this will be necessary, but your 15 out of 15 runs brings some extra confidence.